includes/sales-tax.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace WSU\WooCommerce_Extended\Sales_Tax;
+
+/**
+ * Short circuit database tax table lookup.
+ *
+ * @since 0.1.0
+ */
+add_filter( 'woocommerce_customer_taxable_address', '__return_empty_array', 10 );
+
+add_filter( 'woocommerce_matched_rates', 'WSU\WooCommerce_Extended\Sales_Tax\find_tax_rate' );
+add_filter( 'woocommerce_rate_code', 'WSU\WooCommerce_Extended\Sales_Tax\rate_code', 10, 2 );
+add_filter( 'woocommerce_rate_label', 'WSU\WooCommerce_Extended\Sales_Tax\rate_label', 10, 2 );
+
+/**
+ * Finds the tax rate for a customer's cart address using the
+ * WA DOR sales tax API.
+ *
+ * @since 0.1.0
+ *
+ * @return array
+ */
+function find_tax_rate() {
+	$address = WC()->customer->get_shipping_address_1();
+	$state    = WC()->customer->get_shipping_state();
+	$postcode = WC()->customer->get_shipping_postcode();
+	$city     = WC()->customer->get_shipping_city();
+
+	if ( 'WA' !== $state ) {
+		return array();
+	}
+
+	$matched_rates = array();
+
+	$tax_url = 'http://dor.wa.gov/AddressRates.aspx?output=text';
+	$tax_url = add_query_arg( array(
+		'output' => 'text',
```

Comment:
The `output=text` parameter is already part of `$tax_url` above, though having it twice doesn't seem to hurt anything :)